### DISCUSSION TITLE
writeStatus not working

### DISCUSSION THREAD

**Discussion by codergautam** [uNetworking/uWebSockets.js#743] [General]
![image](https://user-images.githubusercontent.com/66850133/169618457-432ab33a-7627-4722-966a-ef06a7bf2b82.png)
<img width="752" alt="image" src="https://user-images.githubusercontent.com/66850133/169618533-81db9302-a91b-45b8-b3f1-45b42a7633c1.png">

**Comment by e3dio** [comment]
writeStatus before writeHeader, otherwise automatic 200 status

**Reply by codergautam** [reply]
Thanks!

**Reply by e3dio** [reply]
See https://github.com/uNetworking/uWebSockets.js/discussions/665#discussioncomment-1833921 for more example

**Reply by saikumarpb** [reply]
this helped, thank you
